### DISCUSSION TITLE
How can I force VSCode to use the Fixer installed as a project dependency instead of its built-in PHAR?

### DISCUSSION THREAD

**Discussion by Alexandre-Fernandez** [PHP-CS-Fixer/PHP-CS-Fixer#7098] [Q&A]
How can I force the [VSCode's extension](https://github.com/junstyle/vscode-php-cs-fixer) to use Fixer installed as a project dependency instead of its IDE's built-in PHAR?

**Comment by Wirone** [comment]
Hi Alexandre! I would like to help you, but I don't understand your problem. You either install Fixer as dev dependency and use it with `vendor/bin/php-cs-fixer` or download PHAR and run the tool through it. Can you share some more information?

**Reply by Alexandre-Fernandez** [reply]
Sorry I didn't post this is the good repo, I'm using [the VSCode extension](https://github.com/junstyle/vscode-php-cs-fixer) and I want to force it to use the locally installed php-cs-fixer instead of the global one, because the global one may not have the extensions that the local one has.

**Reply by Wirone** [reply]
Sorry, I can't help you with VSCode since I don't use it. This is out of the scope of this repository 🙂.
